ci-operator/config/openshift/backplane-cli/openshift-backplane-cli-main.yaml

```
@@ -28,6 +28,10 @@ tests:
     make test
   container:
     from: src
+- as: scan
+  commands: make scan
+  container:
+    from: src
```

This should be optional as stated earlier.

```suggestion
- as: scan
  commands: make scan
  container:
    from: src
  options: true
```